lib/t/push_notifications/dispatch_job.ex

```diff
@@ -54,8 +53,17 @@ defmodule T.PushNotifications.DispatchJob do
   defp handle_type(type, args) when type in ["timeslot_offer", "timeslot_accepted"] do
     %{"match_id" => match_id, "receiver_id" => receiver_id} = args
 
-    if alive_match(match_id) do
-      if profile = profile_info(receiver_id) do
+    if match = alive_match(match_id) do
+      %Matches.Match{user_id_1: uid1, user_id_2: uid2} = match
+
+      picker_id =
+        if(receiver_id == uid1) do
+          uid2
+        else
+          uid1
+        end
```

Comment:
I would add `picker_id` to the `args` above to declutter the code here
can be done in places like this:
![Uploading image.png…]()

Reply:
good idea, made like this